code/_onclick/hud/new_player.dm

```diff
@@ -348,14 +414,97 @@
 		set_button_status(FALSE)
 		return
 
-/atom/movable/screen/lobby/button/poll/update_overlays()
+/atom/movable/screen/lobby/button/bottom/poll/update_overlays()
 	. = ..()
 	if(new_poll)
 		. += mutable_appearance('icons/hud/lobby/poll_overlay.dmi', "new_poll")
 
-/atom/movable/screen/lobby/button/poll/Click(location, control, params)
+/atom/movable/screen/lobby/button/bottom/poll/Click(location, control, params)
 	. = ..()
 	if(!.)
 		return
 	var/mob/dead/new_player/new_player = hud.mymob
 	new_player.handle_player_polling()
+
+/atom/movable/screen/lobby/button/collapse
+	name = "Collapse Lobby Menu"
+	icon = 'icons/hud/lobby/collapse_expand.dmi'
+	icon_state = "collapse"
+	base_icon_state = "collapse"
+	layer = LOBBY_BELOW_MENU_LAYER
+	screen_loc = "TOP:-82,CENTER:-54"
+	always_shown = TRUE
+
+/atom/movable/screen/lobby/button/collapse/Click(location, control, params)
+	. = ..()
+	if(!.)
+		return
+	var/mob/dead/new_player/new_player = hud.mymob
+	if(!istype(hud, /datum/hud/new_player))
+		return
+	var/datum/hud/new_player/our_hud = hud
+	base_icon_state = our_hud.menu_hud_status ? "expand" : "collapse"
+	name = "[our_hud.menu_hud_status ? "Expand" : "Collapse"] Lobby Menu"
+	set_button_status(FALSE)
+	//re-enable clicking the button when the shutter animation finishes
+	addtimer(CALLBACK(src, PROC_REF(set_button_status), TRUE), (2 * SHUTTER_MOVEMENT_DURATION + SHUTTER_WAIT_DURATION), TIMER_CLIENT_TIME)
+
+	//get the shutter object used by our hud
+	var/atom/movable/screen/lobby/shutter/menu_shutter = locate(/atom/movable/screen/lobby/shutter) in hud.static_inventory
+
+	addtimer(CALLBACK(src, PROC_REF(toggle_menu), new_player, our_hud), SHUTTER_MOVEMENT_DURATION + 1, TIMER_CLIENT_TIME) //ever so slightly after the shutter pulls down
+	//animate the shutter
+	menu_shutter.setup_shutter_animation()
+	//animate bottom buttons' movement
+	if(our_hud.menu_hud_status)
+		collapse_buttons()
+	else
+		expand_buttons()
+
+///Moves the button to the top of the screen, leaving only the screen part in view
+///Sends a signal on the hud for the bottom buttons to listen to
+/atom/movable/screen/lobby/button/collapse/proc/collapse_buttons()
+	SEND_SIGNAL(hud, COMSIG_HUD_LOBBY_COLLAPSED)
+	var/matrix/transformation = matrix()
+	//wait for the shutter to come down
+	animate(src, transform = transformation, time = SHUTTER_MOVEMENT_DURATION + SHUTTER_WAIT_DURATION)
+	//then pull the button up with the shutter and leave it on the edge of the screen
+	animate(transform = transformation.Translate(x = 0, y = 134), time = SHUTTER_MOVEMENT_DURATION, easing = CUBIC_EASING|EASE_IN)
```

Comment:
similar question about the var here

Reply:
https://github.com/tgstation/tgstation/pull/76443/commits/9735e710e838bb6be51edc9c03dbe0e9688a8a48